### DISCUSSION TITLE
How to use an event like "Marriage"

### DISCUSSION THREAD

**Discussion by Biepa** [gramps-project/gramps-web#509] [Q&A]
Hi, first time user of this project.
I logged in, created two users and wanted to connect them via a marriage event. While I could create the event, it's not obvious how to add the persons that married. 
Is there documentation I missed? I looked for "events" but couldn't find anything.

**Comment by DavidMStraub** [comment]
Hi! At the moment, the way to do it is to click the "+" button on the top right and select "Add: Family". Then you can add the existing persons as father and mother.

In Gramps (Web or Desktop), the marriage is an _Event_ attached to the _Family_, which is connects people.

Obviously this is not very intuitive at first and there is an open feature request to be able to add a person to a family directly form the person view. https://github.com/gramps-project/gramps-web/issues/227

**Reply by Biepa** [reply]
Thank you @DavidMStraub
